docs/dapps/getting-started.md

```diff
@@ -8,91 +8,531 @@ import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
 
 <UntranslatedPageText />
 
+#Dapps Getting started
+
 This guide will explore the basics of creating an Alephium dApp project.
 
 Prerequisites:
 
 - Write code in [Typescript](https://www.typescriptlang.org/)
 - Operate in a [terminal](https://en.wikipedia.org/wiki/Terminal_emulator)
```

Comment:
I think it would be good to mention somewhere that `node`, `npm`, and `npx` need to be installed in the system, as well as their versions (at least for `node` and `npm`).

Reply:
sure